crates/tabby-common/src/lib.rs

```diff
@@ -48,6 +48,12 @@ impl SourceFile {
     }
 }
 
+pub fn validate_identifier(ident: &str) -> bool {
```

Comment:
move to a static method of `RepositoryConfig`, e.g `RepositoryConfig::is_valid_name`